### ISSUE TITLE
[BUG] - Carousel: "skip-entries" link not visible on `:focus-visible` 

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3050]
### JS Framework

All

### Porsche Design System version

3.12.0

### Reproduction Example

https://stackblitz.com/edit/react-gitozp?file=App.tsx

### Steps to reproduce the behavior

1. Go to [Stackblitz](https://stackblitz.com/edit/react-gitozp?file=App.tsx)
2. Press Tab-Key until the prev/next buttons gets focus.
3. See error: the tab before the "prev" button focuses the "skip-entries" link but it is still hidden

### Expected behavior

The "skip-entries" link should become visible on `:focus-visible` 

### Notes

- Somehow `:focus-visible` is not getting applied on `.skip-link` although custom element should be the `activeElement`.
- Seams like Skip Link behaviour is not yet covered by test automation?
